src/components/EventsList/index.tsx

```diff
@@ -1,14 +1,30 @@
 import styled from "@emotion/styled";
+import { useRouter } from "next/router";
+import { useState } from "react";
+
+interface categoryProps {
+  curCategory: number;
+}
 
 export default function EventsList() {
+  const [curCategory, setCurCategory] = useState(1);
```

Comment:
T2:  타입스크립트를 쓰는 만큼  타입 추론보다는 타입을 명시해서 지윤님처럼 적어주시면 좋을 것 같습니다.
```suggestion
  const [curCategory, setCurCategory] = useState<number>(1);
```